docs: update readme (#16)
edx/docs.edx.org: README.md

## Overview

> This is the Github-hosted and tied to the master branch that is behind

Reviewer: ```suggestion
This is the Github Pages-hosted repo behind
```

http://docs.edx.org. 

## Search

The `search.html` page is powered by Google Programmable Search Engine and can be configured [here](https://cse.google.com/cse/setup/basic?cx=003935656615667250026:jtodz7s8c9a):

The current list of sites that should be configured there is in `search-sites.txt`.

## Release

> There is no extra release process and as soon as you merge to master, the changes will be reflected staright away on production. 

Reviewer: ```suggestion
There is no extra release process and as soon as you merge to master, the changes will be reflected straight away on production. 
```

Author: good catch.

## Overview
This is the Github-hosted and tied to the master branch that is behind
http://docs.edx.org. 

## Search

The `search.html` page is powered by Google Programmable Search Engine and can be configured [here](https://cse.google.com/cse/setup/basic?cx=003935656615667250026:jtodz7s8c9a):

The current list of sites that should be configured there is in `search-sites.txt`.

> ## Contribution

Reviewer 2: I like the idea, but this should probably be in `.github/CONTRIBUTORS.md` (see some other edX repos for examples). This is a more standard and GitHub-integrated way of surfacing that info.

Author: I think github talks about having that file at the root, not in `.github/` can you confirm? 
https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors

Reviewer 2: ^ The doc allows for that file in root, `docs`, or `.github`. `.github` is my personal preference but it could be in any of those 3.

Author: yes, right :)

> If you want to contibute, please open a pull request against master and tag [community-engineering](https://github.com/orgs/edx/teams/community-engineering) team for a review. There is no extra release process and as soon as you merge to master, the changes will be reflected staright away on production. 

Reviewer 3: Can people who are not edX employees see that team information? Is there an email address they can reach out to?

Reviewer 2: I don't think teams are public. At least hitting it when logged out of GitHub doesn't show up.

Reviewer 2: That said, they are a codeowner so should be auto-added to PRs, yes?

Author: Good call, let me check. 
if they are not public, how we should reword it to indicate that they are the owners and should be tagged for the changes?

Author: Yes, 404 for non-edx team member.

Author: I have removed the _team_ reference/link from the document now.
